types/fparser/test/fparser-tests.ts

```diff
@@ -0,0 +1,7 @@
+import Formula from "fparser";
```

Comment:
```suggestion
import Formula = require('fparser');
```
this would work, once you update `index.d.ts`.
`import Formula from 'fparser'` 'will work when consuming 'fparser' and types, when using modern TS config defaults, which is not a default setup on DT repo, to promote wider interoperability